treccovid_semantic_search/index.json

```diff
@@ -0,0 +1,30 @@
+{
+  "settings": {
+    "index.number_of_shards": {{number_of_shards | default(1)}},
```

Comment:
lets set this value only if it is defined

Reply:
Done, no default num of shards.